From 94b894c3f66d2d593115b861d7ba510c277f0037 Mon Sep 17 00:00:00 2001
From: Abhishek Kumar <abhishek4.kumar@intel.com>
Date: Wed, 27 Jan 2021 21:26:23 +0530
Subject: [PATCH 2/2] intel: change urb max shader geometry for CML GT1

Below deqp cts failure is seen on CML GT1 only , GT2 all test
passes, changing the max shader geometry to 256 (previous 640)
fixes all failure tests.Similar issue on BDW GT1
https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/3173

dEQP-GLES31.functional.geometry_shading.layered.
                render_with_default_layer_cubemap
                render_with_default_layer_3d
                render_with_default_layer_2d_array

Closes: https://gitlab.freedesktop.org/mesa/mesa/-/issues/4102

Signed-off-by: Abhishek Kumar <abhishek4.kumar@intel.com>
Acked-by: Lionel Landwerlin <lionel.g.landwerlin@intel.com>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/8550>
(cherry picked from commit 276e3927b54292b2c0687b1f9bbcc0d69e314de7)
---
 src/intel/dev/gen_device_info.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/intel/dev/gen_device_info.c b/src/intel/dev/gen_device_info.c
index 1ae1207f517e..c325f4276179 100644
--- a/src/intel/dev/gen_device_info.c
+++ b/src/intel/dev/gen_device_info.c
@@ -785,6 +785,7 @@ static const struct gen_device_info gen_device_info_cfl_gt1 = {
    .simulator_id = 24,
    .urb.size = 128,
    .urb.max_entries[MESA_SHADER_VERTEX] = 928,
+   .urb.max_entries[MESA_SHADER_GEOMETRY] = 256,
 };
 
 static const struct gen_device_info gen_device_info_cfl_gt2 = {
-- 
2.29.2

